SKILL.tsv

```diff
@@ -3708,7 +3708,7 @@ SKILL_20150414_003707	{memo X}Enlarge size of target. Target has chance of explo
 SKILL_20150414_003708	Change your INT and CON.
 SKILL_20150414_003709	Control enemy's magic circle.
 SKILL_20150414_003710	{memo X}Enlarge the size of your and party members' hand holding shield and sub weapon to increase defense.
-SKILL_20150414_003711	Enlarge the size of your and party members' head to increase defense.
+SKILL_20150414_003711	Enlarge the size of your and party members' head to increase INT.
```

Comment:
Improves/Increases the INT of you and your party members by enlarging the size of the head. 

(something like that, because 'your and party members' sounds odd)